lib/ansible/modules/storage/purestorage/_purefb_facts.py

```diff
@@ -69,7 +69,8 @@
   description: Returns the facts collected from the FlashBlade
   returned: always
   type: complex
-  contains:
+  sample: >
```

Comment:
Do we need the `>` here? If not, remove from several other sample sections below as well.

Reply:
In this case, it was necessary since the content is no valid JSON or YAML. I've now fixed it in 63381ca578f09fd860bfa5a46604d202109f159f so sanity checks will now pass despite `>` being removed. :)